package org.sas04225.proto;

option java_package = "org.sas04225.proto";
option java_outer_classname = "ImageGroupProto";

message ImageGroup
{
	required string group_name = 1;
	repeated string images = 2;
}
